src/repositories/UserRepository.ts

```diff
@@ -23,6 +23,8 @@ export class UserRepository extends AbstractRepository<UserModel> {
   public async getUserByGoogleId(googleId: Uuid): Promise<UserModel | undefined> {
     return await this.repository
       .createQueryBuilder("user")
+      .leftJoin("user.saved", "post")
+      .leftJoinAndSelect("user.saved", "postSelect")
```

Comment:
We do not want to return a user's saved posts anytime we get a user. Jessie is having me implement a separate route to get a user's saved posts but does not want to always be returning it in our other routes

Reply:
I believe this will be addressed in your next PR

Reply:
^.^